test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -12183,7 +12183,7 @@ def parse_resource_group(self, hostname=None, resource_group=None):
         tree.update()
         return tree
 
-    def add_to_resource_group(self, name, id, parent, nshares):
+    def add_to_resource_group(self, name, id, parent, nshares, validate=True):
         """
```

Comment:
@Mahalty Consider adding/editing 'fairshare_id' and 'validate' to the docstring.